charts/datarepo-api/ci/testing-values.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# overrides an application.properties item in DRmanager
+image:
+  tag: 1.0.180
```

Comment:
How is this tag value going to be kept in sync (same Q for the UI chart)

Reply:
might be worth making an action for updating the chart default so wee dont need to set it.